packages/web/app/src/components/project/alerts/create-channel.tsx

```diff
@@ -42,186 +100,258 @@ export const CreateChannelModal = ({
   organizationId: string;
   projectId: string;
 }): ReactElement => {
-  const [mutation, mutate] = useMutation(CreateChannel_AddAlertChannelMutation);
-  const { errors, values, touched, handleChange, handleBlur, handleSubmit, isSubmitting } =
-    useFormik({
-      initialValues: {
-        name: '',
-        type: '' as AlertChannelType,
-        slackChannel: '',
-        endpoint: '',
-      },
-      validationSchema: Yup.object().shape({
-        name: Yup.string().required('Must enter name'),
-        type: Yup.mixed().oneOf(Object.values(AlertChannelType)).required('Must select type'),
-        slackChannel: Yup.string()
-          .matches(/^[@#]{1}/, 'Must start with a @ or # character')
-          .when('type', ([type], schema) =>
-            type === AlertChannelType.Slack ? schema.required('Must enter slack channel') : schema,
```

Comment:
I think `Slack channel is required` is better wording here. Also in the other errors.